tripal/tripal.module

```diff
@@ -78,6 +78,23 @@ function tripal_theme() {
   return $theme;
 }
 
+/**
+ * Implements hook_library_info().
+ */
+function tripal_library_info() {
+  // @see css/tripal.toolbar.css and miscellaneous.
+  $libraries = [];
+  $toolbar_style = drupal_get_path('module', 'tripal') . '/css/tripal.toolbar.css';
+
+  $lbraries['tripal.icons'] = [
```

Comment:
```suggestion
  $libraries['tripal.icons'] = [
```
Typo. Not sure if this is even needed since the type renders it useless yet the icons seem to be working?